tools/PrepareRelease/SyncMsiContent.cs

```diff
@@ -51,6 +51,9 @@ public static void Run()
             CreateWixFile(
                 groupId: "Files.Managed.NetStandard20",
                 frameworkMoniker: "netstandard2.0");
+            CreateWixFile(
+                groupId: "Files.Managed.Netcoreapp3.1",
```

Comment:
I think the period between 3 and 1 should be removed to keep in sync with the other MSI files that use the name `Files.Managed.Netcoreapp31`

Reply:
Fixed